metadata.yaml

```diff
@@ -38,6 +38,9 @@ resources:
     type: oci-image
     description: OCI image for Jenkins
 requires:
-  agent:
+  slave:
     interface: jenkins-slave
     optional: true
+  agent:
+    interface: jenkins_agent_v0
```

Comment:
This is the first time I've seen a version number appended to a relation. It's an interesting pattern, wonder if it's been discussed? Another approach could be for instance to include a version in the relation data.